src/Sulu/Bundle/ContentBundle/Resources/public/js/extensions/seo-tab.js

```diff
@@ -0,0 +1,222 @@
+
+                // update description
+                this.sandbox.dom.html(
+                    this.$find('#seo-excerpt-description'),
+                    this.sandbox.dom.val(this.$find('#seo-description'))
+                );
+            },
+
+            initializeTitleCounter: function() {
+                this.title.$el = this.$find('#seo-title');
+                this.title.$counter = this.$find('#title-left');
+                this.updateTitleCounter();
+                this.sandbox.dom.on(this.title.$el, 'keyup', this.updateTitleCounter.bind(this));
+            },
+
+            updateTitleCounter: function() {
+                var charsLeft = this.options.maxTitleCharacters - this.sandbox.dom.val(this.title.$el).length;
+                this.sandbox.dom.html(this.title.$counter, ' ' + charsLeft + ' ');
+                this.toggleWarning(this.title.$counter.parent(), (charsLeft <= 0));
+            },
+
+            initializeDescriptionCounter: function() {
+                this.description.$el = this.$find('#seo-description');
+                this.description.$counter = this.$find('#description-left');
+                this.updateDescriptionCounter();
+                this.sandbox.dom.on(this.description.$el, 'keyup', this.updateDescriptionCounter.bind(this));
+            },
+
+            updateDescriptionCounter: function() {
+                var charsLeft = this.options.maxDescriptionCharacters - this.sandbox.dom.val(this.description.$el).length;
+                this.sandbox.dom.html(this.description.$counter, ' ' + charsLeft + ' ');
+                this.toggleWarning(this.description.$counter.parent(), (charsLeft <= 0));
+            },
+
+            updateKeywordsCounter: function() {
+                var value = this.sandbox.dom.trim(
+                    this.sandbox.dom.trim(this.sandbox.dom.val(this.keywords.$el)),
+                    this.options.keywordsSeparator
+                    ),
+                    keywords = value.split(this.options.keywordsSeparator),
+                    keywordsLeft = this.options.maxKeywords;
+                // remove empty entries
+                keywords = keywords.filter(function(value) {
+                    return !!value;
+                });
+                this.keywords.count = keywords.length;
+                keywordsLeft = keywordsLeft - this.keywords.count;
+                this.sandbox.dom.html(this.keywords.$counter, keywordsLeft);
+                this.toggleWarning(this.keywords.$counter.parent(), (keywordsLeft < 0));
+            },
+
+            listenForChange: function() {
+                this.sandbox.dom.on(formId, 'keyup change', function() {
+                    this.setHeaderBar();
+                }.bind(this), '.trigger-save-button');
+            },
+
+            setHeaderBar: function() {
+                this.sandbox.emit('sulu.tab.dirty');
```

Comment:
yes in other usages (if the new component structure will be used) this method don`t have to be overwritten. i will add documentation in this file.